101-VM-Virus-Attack-Prevention/nested/microsoft.network/publicipaddress.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "publicIPAddressName": {
+            "type": "string"
+        },
+        "publicIPAddressType": {
+            "type": "string",
+            "defaultValue": "Dynamic"
+        },
+        "dnsNameForPublicIP": {
+            "type": "string",
+            "defaultValue": "[parameters('publicIPAddressName')]"
```

Comment:
remove this since it's not globally unique

Reply:
done. Thanks !